patches/minecraft/net/minecraft/world/level/levelgen/PhantomSpawner.java.patch

```diff
@@ -0,0 +1,28 @@
+--- a/net/minecraft/world/level/levelgen/PhantomSpawner.java
++++ b/net/minecraft/world/level/levelgen/PhantomSpawner.java
+@@ -42,19 +_,21 @@
+                for(ServerPlayer serverplayer : p_64576_.m_6907_()) {
+                   if (!serverplayer.m_5833_()) {
+                      BlockPos blockpos = serverplayer.m_20183_();
+-                     if (!p_64576_.m_6042_().f_223549_() || blockpos.m_123342_() >= p_64576_.m_5736_() && p_64576_.m_45527_(blockpos)) {
+-                        DifficultyInstance difficultyinstance = p_64576_.m_6436_(blockpos);
++                     DifficultyInstance difficultyinstance = p_64576_.m_6436_(blockpos);
```

Comment:
@LexManos what's your stance on patches like this where only the whitespace is changed?

Reply:
We avoid it wherever possible so that the diff shows the intention of the actual change not the structure of it. We have checks for whitespace changes but this one gets through because its in the same hunk as other functional changes. Its no a show stopper but it is nicer to not change indenting.